Create-Login-and-Register-System-with-C#-Winforms/index.md

```diff
@@ -0,0 +1,430 @@
+### Introduction
+
+C# or C-sharp is an object-oriented programming language developed by Microsoft that runs on a framework called .Net framework. The language has been used to develop many kinds of applications including web applications, mobile applications, computer games, desktop applications, database applications, virtual reality applications, and many others.
+
+### Prerequisites
+
+* A code editor e.g [Visual Studio](https://visualstudio.microsoft.com/vs/).
+* Knowledge of C#, SQL.
+* Running relational database management in this case we will use [MySQL](https://www.mysql.com/)
+* Visual Studio Knowledge
+
+### Step I: Create a database and table with required columns
+
+``` sql
+
+Create database userdata;
+```
+
+![DATABASE CREATION]((/Create-Login-and-Register-System-with-C#-Winforms/create-db-cmd.png))
+
+Create table command
+
+``` sql
+
+CREATE TABLE `user_info` ( `id` INT NOT NULL AUTO_INCREMENT , `names` VARCHAR(20) NOT NULL , `username` VARCHAR(10) NOT NULL , `password` VARCHAR NOT NULL , PRIMARY KEY (`id`)) ;
+```
```

Comment:
This SQL might lead to problems, in case a newbie copy-pastes it. For instance, my full name is "Miller William Juma", someone else will have a longer name than mine, make lengths more accommodating. Most hashed passwords are 60 digits long, specify the length. Usernames should be unique.